gui-tk/library/tls.tcl

```diff
@@ -0,0 +1,398 @@
+#
+# Copyright (C) 1997-2000 Matt Newman <matt@novadigm.com> 
+#
+namespace eval tls {
```

Comment:
Do we still need this file?
Also no clear statement of license.

Reply:
No, I've removed it, thanks.